dataactcore/migrations/versions/fc6be41471a3_populate_submission_reporting_period_.py

```diff
@@ -12,56 +12,16 @@
 branch_labels = None
 depends_on = None
 
-from alembic import op
```

Comment:
Is this revision still necessary even though it's just passing? Would it be problematic to remove it altogether?